jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/BsmCoder.java

```diff
@@ -28,10 +28,10 @@ public Asn1Object decode(InputStream is) {
     }
 
     @Override
-    public void publish(String topic, Asn1Object msg) {
+    public void publishRaw(Asn1Object msg) {

```

Comment:
By the same token, this method should be called simply publish or publishAsn1Object